Description: Source code annotation for animal-sniffer which detects
 and warns about backwards Java compatibility. Not used in Debian build
 of Jenkins and under separate ITP (at which point this patch can be 
 dropped). This patch plugs in the IgnoreJRERequirement annotation to 
 allow compilation only.
Author: James Page <james.page@ubuntu.com>
Forwarded: not-needed

Index: jenkins/core/src/main/java/org/jvnet/animal_sniffer/IgnoreJRERequirement.java
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ jenkins/core/src/main/java/org/jvnet/animal_sniffer/IgnoreJRERequirement.java	2011-11-22 08:20:32.023226979 +0000
@@ -0,0 +1,16 @@
+package org.jvnet.animal_sniffer;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Documented;
+import java.lang.annotation.Target;
+import java.lang.annotation.ElementType;
+import static java.lang.annotation.RetentionPolicy.CLASS;
+
+/**
+ * @author Kohsuke Kawaguchi
+ */
+@Retention(CLASS)
+@Documented
+@Target({ElementType.METHOD, ElementType.CONSTRUCTOR, ElementType.TYPE})
+public @interface IgnoreJRERequirement {
+}
